python/BioSimSpace/Sandpit/Exscientia/FreeEnergy/_restraint.py

```diff
@@ -416,19 +421,27 @@ def format_bond(equilibrium_values, force_constants):
             )
 
         # Format the parameters for the angles and dihedrals
-        def format_angle(equilibrium_values, force_constants):
+        def format_angle(equilibrium_values, force_constants, restraint_lambda):
```

Comment:
It would be good to put some docstrings on these functions because I have no idea why you need them